lib/core/src/lib/datatable/components/columns-selector/columns-selector.component.html

```diff
@@ -33,6 +34,7 @@
             class="adf-columns-selector-search-input"
             data-automation-id="adf-columns-selector-search-input"
             type="text"
+            role="menuitem"
```

Comment:
I think this change causes new accessibility issue in axe tool: 
<img width="1119" alt="image" src="https://github.com/user-attachments/assets/20f78847-a4b3-4610-bc04-2af6394607fd" />

Reply:
hmm, I don't see this one 
![Screenshot 2025-02-21 at 13 31 32](https://github.com/user-attachments/assets/9009e374-3ffc-4e0b-b9c2-dc34ef39aafd)

Reply:
I see it now, it was hidden by Best Practices flag, but removing role="menuitem" from this input will result to the following issue which is marked as critical impact, while the one you've attached has minor impact 
![Screenshot 2025-02-21 at 13 39 14](https://github.com/user-attachments/assets/34037539-55ae-43b4-acab-e9b348c122dd)

Reply:
I think it may be because I see on your screenshot that you have set off Best Practices option in axe. Please set on this option and then you should see that issue @nikita-web-ua

Reply:
@nikita-web-ua ok, I got it. Could you please check if there is any way to resolve both issues? If there is no way to fix both then please just add comment on your jira that this fix introduced minor issue and document that we can't fix it because it introduces critical issue. Is that fine?

Reply:
added comment to the ticket

Reply:
thank you